srcpkgs/AppStream/template

```diff
@@ -1,48 +1,94 @@
 # Template file for 'AppStream'
 pkgname=AppStream
-version=0.16.4
+version=1.0.2
 revision=1
 build_style=meson
 build_helper="gir qemu"
 configure_args="$(vopt_bool gir gir) $(vopt_bool vala vapi)
- -Dstemming=false -Dapidocs=false -Dqt=true -Dsystemd=false"
+ -Dstemming=false -Dapidocs=false -Dqt=true -Dsystemd=false
+ -Dcompose=true"
 hostmakedepends="docbook-xsl gettext glib-devel gperf libxslt pkg-config
- qt5-host-tools $(vopt_if vala vala) itstool"
-makedepends="libglib-devel libxml2-devel libyaml-devel qt5-devel libxmlb-devel
- libcurl-devel"
+ qt6-base $(vopt_if vala vala) itstool"
+makedepends="libglib-devel libxml2-devel libyaml-devel qt6-base-devel libxmlb-devel
+ libcurl-devel librsvg-devel"
+checkdepends="tar"
 short_desc="Tools and libraries to work with AppStream metadata"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/ximion/appstream"
 changelog="https://raw.githubusercontent.com/ximion/appstream/master/NEWS"
 distfiles="https://github.com/ximion/appstream/archive/v${version}.tar.gz"
-checksum=95d5cf451d1945182a9bc4d597c13e713451a3dba1a5759f45b6b3279ff3774c
-conf_files="/etc/appstream.conf"
+checksum=77e271f47167ae37a68111b951c3c07e4261579e69047747044f7924c8219d14
 triggers="appstream-cache"
 
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" AppStream-devel"
+fi
+
 build_options="gir vala"
 build_options_default="gir vala"
 
+pre_configure() {
+	# Meson uses paths from pkg-config which is /usr/lib32/qt6 when cross compiling 32-bit
+	# Work around this for now
+	export PATH="/usr/lib/qt6/libexec/:$PATH"
+}
+
 AppStream-devel_package() {
-	depends="${sourcepkg}>=${version}_${revision} AppStream-qt>=${version}_${revision}"
+	depends="${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
-		vmove usr/include
-		vmove usr/lib/cmake
-		vmove usr/lib/pkgconfig
-		vmove "usr/lib/*.so"
+		vmove usr/include/appstream
+		vmove usr/lib/pkgconfig/appstream.pc
+		vmove /usr/lib/libappstream.so
 		if [ "$build_option_gir" ]; then
-			vmove usr/share/gir-1.0
+			vmove usr/share/gir-1.0/AppStream-1.0.gir
 		fi
 		if [ "$build_option_vala" ]; then
-			vmove usr/share/vala
+			vmove usr/share/vala/vapi/appstream.*
+		fi
+	}
+}
+
+AppStream-compose_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - compose libraries and tools"
+	pkg_install() {
+		vmove usr/libexec/appstreamcli-compose
+		vmove usr/share/man/man1/appstreamcli-compose.1*
```

Comment:
Ideally all wildcards should be quoted, otherwise expansion is even more nondeterministic

Reply:
Thanks for the reminder.